framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -30,15 +34,24 @@ export const ComboPicker = ({
   const buttonRef = useRef<any>()
   const selectRef = useRef<any>()
 
-  const getNewValue = (option: ComboPickerOption, input?: number): ComboPickerValue => ((input)
-    ? { input: Number(input), option }
-    : { option })
+  const getNewValue = (option: ComboPickerOption, input?: number): ComboPickerValue => (
+    (is(Number, input))
+      ? { input: Number(input), option }
+      : { option })
+
+  const getInputWithinRange = ((input: number) => {
+    if (max && input > max) return max
+    if (min && input < min) return min
+
+    return input
+  })
```

Comment:
Default max=Infinity and min=-Infinity and then return clamp(min, max, input), it's a util in northlight

Reply:
Changed